genesyscloud/data_source_genesyscloud_routing_settings_test.go

```diff
@@ -0,0 +1,58 @@
+package genesyscloud
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"strings"
+	"testing"
+)
+
+func TestAccDataSourceRoutingSettings(t *testing.T) {
+	var (
+		settingsResource   = "test-settings"
+		settingsDataSource = "test-settings-data"
+		transcription      = "EnabledQueueFlow"
+		confidence         = "1"
+	)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: providerFactories,
+		Steps: []resource.TestStep{
+			{
+				// Search by contact center
+				Config: generateRoutingSettingsWithCustomAttrs(
+					settingsResource,
+					trueValue,
+					generateSettingsTranscription(transcription, confidence, trueValue, falseValue),
+				) + generateRoutingSettingsDataSource(
+					settingsDataSource,
+					trueValue,
+					"genesyscloud_routing_settings."+settingsResource,
+					generateSettingsTranscription(transcription, confidence, trueValue, falseValue),
+				),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrPair("data.genesyscloud_routing_settings."+settingsDataSource, "transcription.0.transcription", "genesyscloud_routing_settings."+settingsResource, "transcription.0.transcription"),
```

Comment:
Could add a few more checks here for the reset_agent_on_presence_change value and contactcenter values